step-framework-server/src/main/java/step/framework/server/CacheControlFilter.java

```diff
@@ -0,0 +1,38 @@
+package step.framework.server;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.FilterConfig;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletResponse;
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
+
+public class CacheControlFilter implements Filter {
+
+    @Override
+    public void init(FilterConfig filterConfig) {}
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+        if (servletRequest instanceof Request) {
+            Request request = (Request) servletRequest;
+            HttpServletResponse httpResponse = (HttpServletResponse) response;
```

Comment:
I assume it is safe but we could add an explicit type check if we want to be 100% safe

Reply:
It is safe, but I'll add a check for the form.